### ISSUE TITLE
Remove Exif Data

### ISSUE THREAD

**Issue by Nelinski** [SergiX44/XBackBone#247]
**Is your feature request related to a problem? Please describe.**
I'd like an option to automatically remove Exif data for privacy. 

**Describe the solution you'd like**
Automatically remove Exif data when files are uploaded or an option to toggle this in settings. 

**Additional context**
N/A

**Comment by taulen** [comment]
Indeed, I would also love this. I also upload images from my phone to xbackbone, and it includes all exif, gps and everything which isnt optimal if you want to share some photos a bit "wider".
